.builders/images/windows-x86_64/Dockerfile

```diff
@@ -116,6 +117,32 @@ RUN Get-RemoteFile `
 
 ENV OPENSSL_VERSION="3.4.1"
 
+# Nasm
+ENV NASM_VERSION="2.16.03"
+RUN Get-RemoteFile `
+    -Uri https://www.nasm.us/pub/nasm/releasebuilds/$Env:NASM_VERSION/win64/nasm-$Env:NASM_VERSION-win64.zip `
+    -Path "nasm-$Env:NASM_VERSION-win64.zip" `
+    -Hash '3ee4782247bcb874378d02f7eab4e294a84d3d15f3f6ee2de2f47a46aa7226e6' && `
+    7z x "nasm-$Env:NASM_VERSION-win64.zip" -o"C:\nasm" && `
+    Add-ToPath -Append "C:\nasm\nasm-$Env:NASM_VERSION" && `
+    Remove-Item "nasm-$Env:NASM_VERSION-win64.zip"
+# openssl
+RUN Get-RemoteFile `
+    -Uri https://www.openssl.org/source/openssl-$Env:OPENSSL_VERSION.tar.gz `
+    -Path openssl-$Env:OPENSSL_VERSION.tar.gz `
+    -Hash '002a2d6b30b58bf4bea46c43bdd96365aaf8daa6c428782aa4feee06da197df3'; `
+    7z x openssl-$Env:OPENSSL_VERSION.tar.gz -r -y && `
+    7z x openssl-$Env:OPENSSL_VERSION.tar -oC:\openssl3-src && `
+    cd C:\openssl3-src\openssl-$Env:OPENSSL_VERSION && `
+    RunOnVSConsole -Command `
+    'C:\perl\perl\bin\perl.exe Configure `
+    "--prefix=C:\openssl" `
+    "--libdir=lib" `
+    no-module no-comp no-idea no-mdc2 no-rc5 no-ssl3 no-gost`
+    && `
+    nmake && `
+    nmake install_sw'
```

Comment:
nit - do we remove the open tar gz file?

Reply:
Yeah I guess we could do that for consistency (in practice it doesn't matter a whole lot, though).